api/src/main/java/io/strimzi/api/kafka/model/kafka/listener/GenericKafkaListenerConfiguration.java

```diff
@@ -58,6 +58,7 @@ public class GenericKafkaListenerConfiguration implements UnknownPropertyPreserv
     private String hostTemplate;
     private String advertisedHostTemplate;
     private Map<String, Object> additionalProperties;
+    private Boolean allocateLoadBalancerNodePorts = true;
```

Comment:
If we don't leave this as `null`, should it be `boolean` instead? Or maybe you should keep it as `Boolean`, have it default to `null` and then in the code check if it is set which might be better approach.

Reply:
Thanks for the review. Yes I will keep it null and change the generate functions to check for null.